plugins/modules/zos_apf.py

```diff
@@ -340,11 +341,15 @@ def backupOper(module, src, backup, tmphlq=None):
     fail_json
         Creating backup has failed.
     """
-    # analysis the file type
-    ds_utils = data_set.DataSetUtils(src)
-    file_type = ds_utils.ds_type()
+    file_type = None
+    if data_set.is_data_set(src):
+        file_type = data_set.DataSet.data_set_type(src)
+    else:
+        if os.path.exists(src):
+            file_type = 'USS'
+
     if file_type != 'USS' and file_type not in DS_TYPE:
-        message = "{0} data set type is NOT supported".format(str(file_type))
+        message = "{0} data set type of {1} is NOT supported".format(str(file_type), src)
```

Comment:
i think your format args _might_ be out of order:
- {0} --> `str(file_type)`. Say it resolves to VSAM,
- {1} --> `src`. Say it resolves `MY.VSAM.DATASET`, 

then the error message would be: 
> VSAM data set type of MY.VSAM.DATASET is NOT supported.

I'm guessing the intended error message would be:
> MY.VSAM.DATASET of type VSAM is NOT supported.